FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/mbedtls/mbedtls_freertos_port.c

```diff
@@ -101,19 +149,35 @@ void mbedtls_platform_mutex_init( mbedtls_threading_mutex_t * pMutex )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Frees a mutex.
+ *
+ * @param[in] pMutex mbedtls mutex handle.
+ *
+ * @note This function is an empty stub as nothing needs to be done to free
+ * a statically allocated FreeRTOS mutex.
+ */
 void mbedtls_platform_mutex_free( mbedtls_threading_mutex_t * pMutex )
 {
-    /* Nothing needs to be done to free a statically-allocated FreeRTOS mutex.
-     */
+    /* Nothing needs to be done to free a statically-allocated FreeRTOS mutex. */
     ( void ) pMutex;
 }
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Function to lock a mutex.
+ *
+ * @param[in] pMutex mbedtls mutex handle.
+ *
+ * @return 0 is always returned as any other failure is asserted.
```

Comment:
(Nit) for this and the unlock function
```suggestion
 * @return 0 (success) is always returned as any other failure is asserted.
```